tests/test_serializers.py

```diff
@@ -72,3 +71,19 @@ def test_taggit_removes_tags(self):
 
         assert TestModel.objects.filter(tags__name__in=["1"]).count() == 0
         assert TestModel.objects.filter(tags__name__in=["1", "2"]).count() == 1
+
+    def test_returns_new_data_after_update(self):
+        """
+        Test if the the prefetch cache clears after an updates
+        """
+        TestModel.objects.create().tags.add("1")
+
+        test_model = TestModel.objects.prefetch_related("tags").get()
```

Comment:
Reading this test at first it wasn't clear what we were testing. 

I think it would be clearer if at first we add an assert here to show that the serializer data is `["1"]`. (`assert TestModelSerializer(test_model).data["tags"] == ["1"]`).

Then at the end of this we can do `set(serializer.data["tags"]) == {"2", "3"}`.